index.bs

```diff
@@ -919,24 +910,37 @@ NOTE: The {{XRRay}}'s {{XRRay/matrix}} can be used to easily position graphical
 Pose {#pose}
 ====
 
-XRViewerPose {#xrviewerpose-interface}
--------------
+XRPose {#xrpose-interface}
+------
 
-An {{XRViewerPose}} describes the state of a <dfn>viewer</dfn> of the XR scene as tracked by the [=XR device=]. A [=viewer=] may represent a tracked piece of hardware, the observed position of a users head relative to the hardware, or some other means of computing a series of viewpoints into the XR scene. The {{XRViewerPose}} describes the position and orientation of the [=viewer=] relative to the {{XRReferenceSpace}} it was queried with, as well as an array of [=view=]s, which include view and projection matrices. These matrices should be used by the application when render a frame of an XR scene.
+An {{XRPose}} describes a position and orientation in space relative to an {{XRSpace}}.
 
 <pre class="idl">
-[SecureContext, Exposed=Window] interface XRViewerPose {
+[SecureContext, Exposed=Window] interface XRPose {
   readonly attribute XRRigidTransform transform;
-  readonly attribute FrozenArray&lt;XRView&gt; views;
+  readonly attribute boolean emulatedPosition;
 };
 </pre>
 
-The <dfn attribute for="XRViewerPose">transform</dfn> is the {{XRRigidTransform}} of the [=viewer=] relative to the origin of the {{XRReferenceSpace}} the {{XRViewerPose}} was queried with.
+The <dfn attribute for="XRPose">transform</dfn> attribute describes the {{XRRigidTransform}} between two {{XRSpace}}.
+
+The <dfn attribute for="XRPose">emulatedPosition</dfn> attribute indicates the accuracy of the {{XRPose/transform}}.{{XRRigidTransform/position}} value. {{XRPose/emulatedPosition}} MUST be set to <code>true</code> if the given positional values are software estimations, such as those provided by a neck or arm model. {{XRPose/emulatedPosition}} MUST be set to <code>false</code> if the positional values are based on sensor readings.
```

Comment:
>indicates the accuracy of the

This phrasing has always somewhat bugged me.  There's something about it that implies that there's a gradient of accuracy being communicated.